script/check_changelogs.py

```diff
@@ -34,19 +42,26 @@ def main():
         with open(os.path.join(cl_dir, cl), mode='rb') as f:
             master = shasum(f)
 
+        any_failures = False
         print("**", cl, master, "**")
         for v in VERSIONS:
             if parsed_version <= parse_version(v):
-                print(f"checking {cl} on {v}")
-                url = f"https://raw.githubusercontent.com/elastic/apm-server/{v}/changelogs/{cl}"
+                print("checking {} on {}".format(cl, v))
+                url = "https://raw.githubusercontent.com/elastic/apm-server/{}/{}".format(v, f.name)
                 rsp = requests.get(url)
+                status = "success"
                 if rsp.status_code == 200:
                     h = shasum(io.BytesIO(rsp.content))
                 else:
-                    h = f"error: {rsp.status_code}"
+                    h = "error: {}".format(rsp.status_code)
                 # rsp.raise_for_status()
-                print(h, url, "🔴" if h != master else "✅")
+                if h != master:
+                    status = "failed"
+                    any_failures = True
+                print(h, url, status)
         print()
+        if args.fail_if_errors and any_failures:
```

Comment:
I think it would be ok to make this always happen if there are `any_failures` (aka remove the flag altogether)